index.js

```diff
@@ -70,6 +78,7 @@ function DB (opts) {
             for (var i = 0; i < node.value.v.refs.length; i++) {
```

Comment:
Right. I'll pass the error along to `done()`, which will pass it up to `next()` which will pass it up to the `hyperlog-join` callback.